metadata-ingestion/src/datahub/utilities/logging_manager.py

```diff
@@ -278,3 +278,6 @@ def configure_logging(debug: bool, log_file: Optional[str] = None) -> Iterator[N
 logging.getLogger("snowflake").setLevel(level=logging.WARNING)
 # logging.getLogger("botocore").setLevel(logging.INFO)
 # logging.getLogger("google").setLevel(logging.INFO)
+logging.getLogger("pyodata.client").setLevel(logging.WARNING)
```

Comment:
loggers have hierarchy, so it think this should work

```suggestion
logging.getLogger("pyodata").setLevel(logging.WARNING)
```